### DISCUSSION TITLE
Hybrid fiscal year model

### DISCUSSION THREAD

**Discussion by lesh4wn** [sql-bi/Bravo#765] [Q&A]
My organization operates on a 5-4-4 fiscal year, but the the beginning and the end of the year is always on 1/1 or 12/31. This is a non-traditional methodology but it's what our organization utilizes.

I also want to note that apart from the date that our year begins and ends, each week begins on Sunday.

How can I generate this type of calendar with Bravo.BI?

**Comment by albertospelta** [comment]
You can create a custom calendar in Bravo by cloning an existing calendar and then customizing it according to your organization's specifications. - You can start from https://docs.sqlbi.com/bravo/features/manage-dates/customize-date-template, follow links to related documentation and watch videos.
